### PR TITLE
AO3-6243 Remove condition for comment form to be 'remote' to avoid CSRF token missing error

### PR DESCRIPTION
**Pull request by EchoEkhi**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [ ] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6243

## Purpose

Fix editing a comment without JavaScript results in a session expired auth error.

## Testing Instructions

Follow instructions on JIRA.

## Credit

EchoEkhi (He/Him/{/([A-Z][a-z]*)/g})

### CODE REVIEW

**Comment by sarken** [comment]
Thank you for the pull request! However, I'm going to temporarily close this, since it looks like you're currently over the limit of 5 open PRs. If you could pick another of your Awaiting Review PRs to close until you've had some PRs merged, we'd appreciate it.

**Comment by EchoEkhi** [comment]
Could you please re-open this? I'm not allowed to re-open it apparently.

File: app/views/comments/_comment_form.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <% commentable = @commentable %>
 <% end %>
 <div class="post comment" id="comment_form_for_<%= commentable.id %>">
-  <%= form_for value_for_comment_form(commentable, comment), :remote => !comment.new_record?, :html => {:id => "comment_for_#{commentable.id}"} do |f| %>
+  <%= form_for value_for_comment_form(commentable, comment), :html => {:id => "comment_for_#{commentable.id}"} do |f| %>
```
Comment (sarken) [inline app/views/comments/_comment_form.html.erb:6]: Can you update the syntax here to `html: { id: "comment_for_#{commentable.id}" }`? Then this should be ready to go!

**Review by sarken — CHANGES_REQUESTED** [review]
Just one little syntax change -- the actual fix looks good!

**Comment by sarken** [comment]
I'm not sure _why_, since you didn't touch the locale files, but it looks like you'll need to run `bundle exec i18n-tasks normalize -l en` to make the tests pass on this.

**Comment by tickinginstant** [comment]
I'm not convinced that this approach to fixing the issue is correct, since I believe this disables the remote form in all cases, even when editing a comment with JS enabled (which currently uses JS with a remote form). You might want to look into the `authenticity_token` argument on form helpers to try to resolve the CSRF error without otherwise changing the functionality.

If this approach is correct and the goal is to make it so that updating a comment never uses JS, even when JS is enabled, then you should probably also delete the code for updating a comment using JS, rather than leave the newly unused code lying around:
https://github.com/otwcode/otwarchive/blob/25150893db066858d83609cc52f36abf288d5071/app/controllers/comments_controller.rb#L376
https://github.com/otwcode/otwarchive/blob/25150893db066858d83609cc52f36abf288d5071/app/views/comments/update.js.erb#L1-L11
It would also be good to add some additional QA steps to cover the change in functionality.

**Comment by sarken** [comment]
Thanks for catching that, ticking! I didn't notice the change in behavior at all when I pulled this and tried it out.

**Review by sarken — APPROVED** [review]

**Pull request merged**